build.gradle

```diff
@@ -57,6 +57,14 @@ task release {
 	description = "The task performed when we are performing a release build.  Relies on " +
 			"the fact that subprojects will appropriately define a release task " +
 			"themselves if they have any release-related activities to perform"
+
+	// Releasing with JDK 11 is not supported yet:
+	// - the hibernate-orm-modules tests do not run due to an issue with the ASM version currently used by Gradle
+	doFirst {
+		if ( JavaVersion.current().isJava11Compatible() ) {
```

Comment:
Since we expect the JARs to be built under JDK8 when releasing, maybe you should test that the JDK is exactly 8 here.

Reply:
Done and merged, thanks for the review.